src/core/shape/2d_primitives.js

```diff
@@ -172,6 +172,15 @@ p5.prototype._normalizeArcAngles = (
  * </code>
  * </div>
  *
+ * <div>
+ * <code>
+ * createCanvas(100, 100, WEBGL);
+ * arc(0, 0, 80, 80, 0, PI + QUARTER_PI, PIE, 7);
+ * describe(
+ *   'ellipse with a quarter missing, and the outline is segmented instead of curved'
+ * );
+ * </code>
+ * </div>
```

Comment:
The round 2D primitives have a `detail` parameter, so technically for completeness they could use an example like this to show it in action.

But I think it would distract from useful parameters and examples, and `detail` doesn't seem like a popular feature for users, and `detail` is already documented well in `curveDetail()` and `bezierDetail()` and the 3D primitives.

Maybe this should make it into the list of WEBGL vs P2D differences? Something like:
- **Vertex Count** - When drawing curved shapes in `WEBGL` mode, you can specify how many vertices are drawn with a `detail` parameter.

Reply:
Instead of `Vertex Count` I think we can use p5's existing language and call it `Shape Detail`. 

Also I don't think additional examples detracts and to be honest I didn't realize the round 2d shapes even had that parameter. I think it would be a good addition, and as you note, the 3d shapes include this, so why not 2d as well?

https://p5js.org/reference/#/p5/sphere